transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/TokenGenerator.kt

```diff
@@ -0,0 +1,5 @@
+package com.genesys.cloud.messenger.transport.util
+
+expect object TokenGenerator {
+    fun generate(): String
```

Comment:
I see the `actual` implementations provide a platform-specific UUID and I noticed that we already have a way to generate platform-specific UUID strings in Platform::randomUUID().

Reply:
Used it!